pages/dmp_supporting_info/support_08_preserve_publish.md

```diff
@@ -63,67 +63,72 @@ When deciding where to archive your data, make suere you chose an archive where
 
-Several sub-questions allow to refer to information in other DMP chapters or external databases. Alternatively, it is always possible to enter a free-text answer.
+Several of the sub-questions here let you refer to information in other chapters of your DMP or in external databases. Alternatively, it is always possible to enter a free-text answer.
 
 #### Which metadata and documentation accompanies the dataset?
-Explain to the best of your knoledge what metadata (what metadata standard) is used for your dataset.
+Explain to the best of your knowledge what metadata (what metadata standard) is used for your dataset. Here you may reuse information from the metadata standard question in the 'Data documentation' chapter.
+
 Metadata is structured information about a dataset (or a document). _How to_ structure the information is done in various ways. Different metadata schemas are designed to capture the information, and such a schema is called a metadata standard. A metadata standard may be designed to suit a subject domain or a specific research area. There are, naturally, different information needs to describe a dataset with biological data, compared to a dataset which revolves on social sciences.
 A metadata schema may capture information by the use of controlled vocabularies. That way, information will be standardised and not be disrupted by different persons using different words or typology for same issues. Such a standardisation will preferably make use of ontologies from a subject domain, and metadata standards will thus often be subject specific. The use of controlled vocabularies and ontologies will increase the quality and findability of the archived data.
 Note that you here may reuse the answer from chapter VI: 'Data documentation during the project'.
 <!-- *write something about standards and use of ontologies* -->
 
 #### Where has the data been archived?
-*write something about preservation strategy if you are NOT using a trustworthy repository*
+You should already have identified actual archives in the 'Identify archive(s)' question above. Thus, you may reuse the information you entered there.
+
+As mentioned above, archives to use should be 'trustworthy'. If you for some reason decide to use an archive not qualified as trustworthy, please consider closely what preservation policy the archive follows. Look for information on this in the 'About' section of the archive, and contact the archive if still in doubt.
```

Comment:
```suggestion
As mentioned above, archives to use should be 'trustworthy'. If you for some reason decide to use an archive not qualified as trustworthy, please consider closely what preservation policy the archive follows. Look for information on this in the 'About' section of the archive, and contact the archive if still in doubt.

In case data is not deposited to an archive but should be preserved locally beyond the project period, it is very important to define a preservation plan and who will be responsible.
```